themes/default/layouts/page/pulumi-up.html

```diff
@@ -10,19 +10,15 @@
                     <span class="green-neon-sign-xl">UP</span>
                 </div>
                 <div class="flex justify-center items-center">
-                    <pulumi-audio
-                        paused-text="FEEL THE VIBE"
-                        playing-text="PAUSE THE VIBE"
-                        url="https://www.pulumi.com/uploads/pulumiup-background-loop.mp4"
-                    ></pulumi-audio>
+                    <a href="#main-event" class="btn btn-orange inline-block ml-8 orange-neon">Watch The Annoucements</a>
                 </div>
             </div>
         </div>
         <div class="container mx-auto pt-4 pb-16">
             <section id="main-event">
                 <div class="w-full">
                     <div class="mb-24">
-                        <h2 class="section-title-blue text-center mt-16 md:mt-4">Watch The Annoucements</h2>
+                        <h2 class="text-center my-16 md:my-4">Watch The Annoucements</h2>
                         <div class="rounded shadow-md" style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;">
```

Comment:
To add consistent margin around the video on mobile:

```suggestion
                        <div class="rounded shadow-md mx-4 md:mx-0" style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;">
```